.github/workflows/pr-automation.yml

```
@@ -0,0 +1,186 @@
+  # 변경 파일 기반 라벨 자동 추가
+  auto-label:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Add Labels Based on Changes
+        uses: actions/labeler@v4
```

actionlint (1.7.11)

[error] the runner of "actions/labeler@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)